packages/playwright/src/reporters/html.ts

```diff
@@ -498,6 +498,7 @@ class HtmlBuilder {
     const { step, duration, count } = dedupedStep;
     const result: TestStep = {
       title: step.title,
+      category: step.category,
```

Comment:
We could also just match on `title` and be fine. So if this has any negative side effects, let me know.